### ISSUE TITLE
Update colors for Demo Architecture to match website theme

### ISSUE THREAD

**Issue by svrnm** [open-telemetry/opentelemetry.io#5360]
The mermaid based architecture diagram for the demo uses colors that do not match the colors of the website theme:

https://opentelemetry.io/docs/demo/architecture/

It would be great if someone would update the colors in the underlying mermaid definition to be in line with other graphics like https://opentelemetry.io/img/otel-diagram.svg or https://opentelemetry.io/img/waterfall-trace.svg

Here is where the colors are defined: https://github.com/open-telemetry/opentelemetry.io/blob/main/content/en/docs/demo/architecture.md?plain=1#L62

**Special Note for Outreachy Contributors**: if you want to work on this issue, please make sure that you have completed the small steps and at least one assigned issue as outlined here: https://github.com/open-telemetry/opentelemetry.io/issues/5290

**Comment by Tharanishwaran** [comment]
hi @svrnm can i work on this issue ?

**Comment by svrnm** [comment]
@Tharanishwaran we are currently running the participation phase for [outreachy](https://www.outreachy.org/), and we need to reserve some good first issues for the participants. If you participate in outreachy please take a look in #5290, otherwise I can currently not assign this issue to you. When the participation phase ends, and this issue is still available, I am happy to assign it.

**Comment by Tharanishwaran** [comment]
Thank you @svrnm

**Comment by ladyami** [comment]
@svrnm can I work on this?

**Comment by svrnm** [comment]
> @svrnm can I work on this?

Please complete https://github.com/open-telemetry/opentelemetry.io/issues/5399 first

**Comment by olamideTiana** [comment]
@svrnm Can I help with this?

**Comment by svrnm** [comment]
@olamideTiana yes! Please take a look at the issue description and provide a PR

**Comment by olamideTiana** [comment]
@svrnm I understand what I need to work on now... I'm working on it

**Comment by svrnm** [comment]
stalled
